netlify/vale/vocab.txt

```diff
@@ -263,6 +263,7 @@ kubectl
 kubemci
 kuwahara
 kv
+latest_operator_version
```

Comment:
I'm hoping that once I merge https://github.com/cockroachdb/docs/pull/14533, this will be unnecessary. It can stay for now and I can remove it later.